supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -2,6 +2,18 @@
 [[code-examples]]
 = Code examples
 
+Provide the source language if it is supported by the toolchain. Do not use the Bash source language for terminal commands; when representing a terminal prompt for a root command, it improperly interprets the # character as a comment.
```

Comment:
ASD-STE100, the specification for Simplified Technical English [1] and [2], says regarding semicolons:

> 8.1 | Use all standard English punctuation marks except the semicolon (;).

[1] https://robertobertuol.com/wp-content/uploads/2018/08/ASD-STE100-ISSUE-7.pdf
[2] http://www.asd-ste100.org

Reply:
@stoobie any reason to follow ASD-STE100 and not the IBM usage on semicolons?

Reply:
@cbudz  It's easier to understand, especially for a global audience. I have not yet encountered a situation where I couldn't easily replace a semicolon.

Reply:
@cbudz, @stoobie, I'd avoid semicolons, if at all possible, even though IBM says that they're okay.

Reply:
@IngridT1 @stoobie I agree and would not use them in product documentation. I am, however, hesitant to go down the path to adopting individual punctuation elements  in our guide. In my opinion, minimalism is our standard for breaking down complex sentences to avoid punctuation like semicolons. Also piecemeal adoption of punctuation elements in style would create a rather annoying experience between our supplement and IBM. I'd rather leave guidance like that to IBM.

Reply:
Oh, sure. Let's just focus on IBM. There are already so many places to look for guidance.

Reply:
+1 for not using guidance outside of the IBM style guide since it's not officially what CCS follows.